lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
Promise are supported in the current node LTS (4.5), not sure if it's required to have an additional library like bluebird.